build.sh

```diff
@@ -254,15 +251,17 @@ PackageTests()
     rm -rf $testPackageFolder
     mkdir $testPackageFolder
 
-    find $sourceFolder -path $testSearchPattern -exec cp -r -u -T "{}" $testPackageFolder \;
+    find . -maxdepth 6 -path $testSearchPattern -exec cp -r "{}" $testPackageFolder \;
```

Comment:
I forget why that was there - I stuck it in when I did the donate build script years ago.  This was just reverting a commit that broke the macos packaging and so stopped tests running there.  Will take a look.

Reply:
Had a look at this and otherwise you end up flattening the directory structure.  The `find` command gets all the files and directories matching `*.Test/bin/x86/Release/*` and recursively copies them into `_test`.  If you omit the `-maxdepth 6`, as well as the recursively copied directories you get every single matched file copied into `_test` also.  The assumption is that all tests projects are under `src` and output to `src/[something].Test/bin/x86/Release` which is 5 levels deep and so we'd get a recursive copy of everything in that `Release` folder.

So it's a bit of a bodge but it would only cause grief if someone set up a whole new test project in `src/MyRandomTests/Lidarr.Awesome.Test`